WalletWasabi/Mono/Option.cs

```diff
@@ -300,7 +298,7 @@ private OptionValueType ParsePrototype()
 				string name = Names[i];
 				if (name.Length == 0)
 				{
-					throw new ArgumentException("Empty option names are not supported.", "prototype");
+					throw new ArgumentException("Empty option names are not supported.", nameof(Prototype).ToLower());
```

Comment:
Bacause `ArgumentException` second parameter is the exact name of param, IMO `.ToLower()` is unnecessary even if the original Exception worked like that. These texts are written to the user on the console so changing it won't cause any trouble.

Reply:
In other exception messages `"prototype"` all lower case was used, that is the only reason I used `.ToLower()`.

Reply:
@nopara73 Should we stay with prototype with lower case in ArgumentException? The name of the property is `Prototype` so that would be the correct one but in the original code it is `prototype`.
These messages are written to the console to the user.